checks/system/unix.py

```diff
@@ -72,7 +72,7 @@ def _parse_darwin(self, output):
         for idx, disk in enumerate(disks):
             kb_t, tps, mb_s = map(float, lastline[(3 * idx):(3 * idx) + 3])  # 3 cols at a time
             io[disk] = {
-                'system.io.bytes_per_s': mb_s * 10**6,
+                'system.io.bytes_per_s': mb_s * 2**20,
```

Comment:
mb_s is a float, so that wouldn't work but yeah i agree it would be better otherwise.